### PR TITLE
removes sex and balls

### PR DESCRIPTION
**Pull request by Bibby0110**
pg-13

### CODE REVIEW

File: code/datums/mutations/speech.dm [hunk 1]
```diff
@@ -97,7 +97,7 @@
 		message = replacetext(message," nerd "," smart guy ")
 		message = replacetext(message," moron "," fun person ")
 		message = replacetext(message," IT'S LOOSE "," EVERYTHING IS FINE ")
-		message = replacetext(message," sex "," hug fight ")
+		message = replacetext(message,"  "," hug fight ")
```
Comment (ToasterBiome) [inline code/datums/mutations/speech.dm:100]: lol

File: code/datums/wires/robot.dm [hunk 2]
```diff
@@ -14,7 +14,7 @@
 
 /datum/wires/robot/interactable(mob/user)
 	var/mob/living/silicon/robot/R = holder
-	if(R.wiresexposed)
```
Comment (ToasterBiome) [inline code/datums/wires/robot.dm:17]: LOL

**Comment by Reed0506** [comment]
what the fuck did i just read

**Comment by Yarinoi** [comment]
what the hell is this

**Pull request merged**